static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js

```diff
@@ -3,8 +3,8 @@ define([
 ], function (config) {
     return function () {
         this.id = 'PaidContentVsOutbrain';
-        this.start = '2017-02-22';
-        this.expiry = '2017-03-22';
+        this.start = '2017-02-25';
+        this.expiry = '2017-03-27   ';
```

Comment:
Nit-pick: Spaces, although it seems the eval on the expiry (when it converts it through `Date()`) is stripping these (else I'm pretty sure a test would have failed!)